include/boost/geometry/formulas/spherical.hpp

```diff
@@ -267,6 +267,14 @@ inline result_direct<CT> spherical_direct(CT const& lon1,
 
         result.lon2 = lon1 + lon2 - omg1;
         result.lat2 = lat2;
+
+        // For longitudes close to the antimeridian the result can be out
+        // of range. Therefore normalize.
+        CT const pi = math::pi<CT>();
+        if (result.lon2 > pi)
+            result.lon2 -= 2 * pi;
+        else if (result.lon2 <= -pi)
+            result.lon2 += 2 * pi;
```

Comment:
why not using `math::normalize_longitude` as in `karney_direct`?

Reply:
I was wondering about it but decided to not use this one because it is more lightweight. `math::normalize_longitude` is able to normalize any coordinates no matter how far the value is from the correct range. Here we know that the result will be only slightly off.

Reply:
Yes, that's a good idea. I'll do it.

Reply:
Done. Please let me know whether or not you're fine with the functions.

Reply:
Thanks, it's fine!